src/daffodilDebugger.ts

```diff
@@ -180,12 +181,29 @@ export async function getDebugger(
         return stopDebugging()
       }
 
+      let workspaceFolder = vscode.workspace.workspaceFolders
+        ? vscode.workspace.workspaceFolders[0].uri.fsPath
+        : vscode.Uri.parse('').fsPath
+
+      // Get daffodilDebugger class paths to be added to the debugger
+      let daffodilDebugClasspath = config.daffodilDebugClasspath.includes(
+        '${workspaceFolder}'
+      )
+        ? config.daffodilDebugClasspath.replace(
+            regexp['workspace'],
+            workspaceFolder
+          )
+        : config.daffodilDebugClasspath
+
       // Start debugger in terminal based on scriptName
       let terminal = vscode.window.createTerminal({
         name: artifact.scriptName,
         cwd: `${rootPath}/daffodil-debugger-${daffodilVersion}-${LIB_VERSION}/bin/`,
         hideFromUser: false,
         shellPath: artifact.scriptName,
+        env: {
+          DAFFODIL_DEBUG_CLASSPATH: daffodilDebugClasspath,
```

Comment:
you can add the following above this line for the linter

`// eslint-disable-next-line @typescript-eslint/naming-convention`

Reply:
@jw3 When my other PR for the CI stuff is merged we will not be using `eslint` anymore as we will be using `prettier` and the way we have it configured, at least right now, it does not complain about this sort of thing.